### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by ramazansakin**
- Parent version fix
- SonarLint issues fixed
- NotFoundExceptions added & adaptated.
- Lombok integration

Remaining issues :
- Logstash connection warning ( Warning logstash connections when logstash instance s not ready )
- Errors on services' connection to Eureka before eureka-server not started ( After eureka-server s up, error disappeared )

### CODE REVIEW

File: containerized-accounts/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM openjdk:11-jdk-slim
 MAINTAINER Taner Diler <taner.diler@gmail.com>
-ADD target/containerized-accounts-1.0.0-SNAPSHOT.jar containerized-accounts.jar
-ENTRYPOINT ["java", "-jar", "/containerized-accounts.jar"]
-EXPOSE 2222
\ No newline at end of file
+ADD target/*.jar containerized-accounts.jar
```
Comment (tanerdiler) [inline containerized-accounts/Dockerfile:3]: Bu çok doğru gelmedi bana. birden fazla jar varsa, en sonuncusu belki de bundle olmayan jar'ı içeri renaming yapıp alacak. bu yüzden pom.xml'de <build><finalName>containerized-accounts</finalName></build> tanımını yapmak daha mantıklı.

File: containerized-discovery/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 FROM openjdk:11-jdk-slim
 MAINTAINER Taner Diler <taner.diler@gmail.com>
-ADD target/containerized-discovery-1.0.0-SNAPSHOT.jar containerized-discovery.jar
-ENTRYPOINT ["java", "-jar", "/containerized-discovery.jar"]
+ADD target/*.jar containerized-discovery.jar
```
Comment (tanerdiler) [inline containerized-discovery/Dockerfile:3]: https://github.com/tanerdiler/spring-boot-microservice-eureka-zuul-docker/pull/4/commits/19e320f1d19c41bc3f32abed1df6fd6653762fbc#r492657034

File: containerized-accounts/Dockerfile (hunk 1 above)
Comment (ramazansakin) [reply containerized-accounts/Dockerfile:3]: Yeah, that's right. I just wanted to make it a little simpler. 👍

**Comment by tanerdiler** [comment]
I took some fixes from your MR but as I said before your changes make the project lose the flexibility on configuration such as using docker service name on communication between microservices.

**Pull request closed without merging**